sdk/servicebus/azure-servicebus/samples/sync_samples/sample_code_servicebus.py

```diff
@@ -167,7 +167,7 @@ def example_send_and_receive_sync():
     # [END create_batch_sync]
 
     # [START send_complex_message]
-    message = ServiceBusMessage(
+    message_send = ServiceBusMessage(
```

Comment:
was mypy complaining about the naming of `message`?

Reply:
Yes since we use message in 2 places mypy was confusing the for **message** in received_messages as a ServiceBusMessage instead of a ServiceBusReceivedMessage because message was previously defined here.